bookkeeper-server/pom.xml

```diff
@@ -283,7 +283,7 @@
       </plugin>
       <plugin>
         <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.2.1</version>
+        <version>${maven-assembly-plugin.version}</version>
```

Comment:
@sijie  @zhaijack 
can we drop the assembly plugin from bookkeeper-server ? or it will be a further step ?

Reply:
Thanks @eolivelli .  I thought it would be better to be similar to DL. in DL there are 2 binary provided, one is -all, while the other is like our bookkeeper-server-bin.  By this way, user will choose which is better for their use case.  
I am OK with both way: drop assemble for bookkeeper-server or not.  
@sijie what do you think of it?

Reply:
@zhaijack it is OK for me